blazor/multiselect-dropdown/refreshdataasync.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: RefreshDataAsync in Blazor MultiSelect Dropdown Component | Syncfusion
+description: Checkout and learn here all about virtualization in Syncfusion Blazor MultiSelect Dropdown component and much more.
+platform: Blazor
+control: MultiSelect Dropdown
+documentation: ug
+---
+
+# RefreshDataAsync in Blazor MultiSelect Dropdown Component
+
+The Remote data source is dynamically updated using the **RefreshDataAsync** function. With the aid of this method, update the data source without the need for a [Query](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_Query). This approach is not a necessary query to render a specific data source count. The **URL** link is required to accomplish that. 
```

Comment:
This approach is not a necessary query to render a specific data source count. The **URL** link is required to accomplish that. 
>>>>>>>>>>>>>>>>
Please modify the above sentence